manager/config/manager/kustomization.yaml

```diff
@@ -7,5 +7,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/the-mesh-for-data/manager
+  newName: kind-registry:5000/m4d-system/manager
```

Comment:
why the change?

Reply:
so that the change is stable when we run the integration-tests ... the root problem which should probably be addressed is that our use of kustomize is altering the file `kustomization.yaml`, so to fix it properly we need to rename the file to `kustomization.yaml.in` and make `kustomization.yaml` an ephemeral file (which has an entry in .gitignore) ... but as I am anyway working on a new helm system to replace eventually kustomize when we may not need this change in the end.

Reply:
would it affect applying the default profile or are we setting the image there separately in a patch?